data/data/gcp/network/lb-private.tf

```diff
@@ -7,7 +7,18 @@ resource "google_compute_address" "cluster_ip" {
 resource "google_compute_health_check" "api_internal" {
   name = "${var.cluster_id}-api-internal"
 
+  // CAUTION: the gcp-routes mechanism must be _faster_ than this value:
```

Comment:
Nit,

Can you maybe add docs/dev/gcp readme that details this and point to that file here?

Otherwise this causes noise here...

Reply:
I can put the link into the readme. But I definitely want to keep this warning here. It's too easy to change this and break all the sensitive timing.

Reply:
> I can put the link into the readme. But I definitely want to keep this warning here. It's too easy to change this and break all the sensitive timing.
```
// IMPORTANT: <link>
```

seems like good enough for approvers and  reviewers

Reply:
I do what you suggest. But for the record, I don't think it is clever to move documentation into the readme.

Reply:
There is no readme yet. Feel free to take over this PR and create it. For me this comment is more than adequat.

Reply:
since https://github.com/openshift/installer/pull/3537 merged, can we update this PR

Reply:
I don't see how #3537 is related to my added comment. This gcp specific, about their strange network setup. #3537 gives generic timing advice.